### DISCUSSION TITLE
Use Frigate + 8K Camera + Black Light to try to detect scorpions

### DISCUSSION THREAD

**Discussion by otaviopfonseca** [blakeblackshear/frigate#14947] [Ask A Question]
Hello Guys!
Recently I'm having serious issues with scorpions in my house. I don't know where they are comming from, so I'm doing some experiments with some cameras that I have and I would like to create a model to detect the scorpions by their glow under black light. It is possible to do it using Frigate? I would do something simple, like anything that glows on the black light I would send a message to the MQTT broker to trigger some alert on Home Assistant.

**Comment by NickM-27** [comment]
I am not aware of any models that do this, you'd need to train your own most likely

**Comment by otaviopfonseca** [comment]
Is It possible to load a trained tensorflow model to frigate? I'm taking this as a possibility.

**Reply by NickM-27** [reply]
specifically object detection models, yes

**Reply by otaviopfonseca** [reply]
Is there some tutorial for this? I've searched on the frigate docs but did not find.
